### PR TITLE
update build process to use babel

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

This pull request will update the component package to use babel 7.

## Summary:

Update project pipeline to use babel to create legacy javascript versions of the src code.

## Type of change:

Please delete options that are not relevant.

- [ ] New capability 
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate) 

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._ 

**Thank you for your submission!**<br>
-- Orion Design System Team

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,6 +48,10 @@
   },
   "devDependencies": {
     "@alaskaairux/orion-web-core-style-sheets": "^0.3.511780638",
+    "@babel/cli": "^7.4.3",
+    "@babel/core": "^7.4.3",
+    "@babel/node": "^7.2.2",
+    "@babel/preset-env": "^7.4.3",
```
Comment (gbedardsice) [inline package.json]: You can remove the babel 6 dependencies below: `babel-cli`, `babel-core`, `babel-preset-es2015`.

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": "entry"
+      }
+    ]
+  ],
+  "targets": {
```
Comment (gbedardsice) [inline .babelrc]: This also needs to be moved inside the preset config, next to `useBuiltIns`.

File: package.json (hunk 1 above)
Comment (blackfalcon) [reply package.json]: Yeah, lazy code post update cleanup. Thanks!

File: .babelrc (hunk 2 above)
Comment (blackfalcon) [reply .babelrc]: Yeah, saw that as an issue [here](https://github.com/babel/babel/issues/6854). Forgot where I copied that from, but apparently, that was WRONG!

**Pull request merged**